src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -599,8 +600,18 @@ internal static IEnumerable<SecurityKey> ConcatSigningKeys(TokenValidationParame
         }
 
         // If a string is in IS8061 format, assume a DateTime is in UTC
+        // Because this is a friend class, we can't remove this method without
+        // breaking compatibility.
         internal static string GetStringClaimValueType(string str)
         {
+            return GetStringClaimValueType(string.Empty, str);
+        }
+
+        internal static string GetStringClaimValueType(string claimType, string str)
```

Comment:
Revert the order to match the above overload.
```suggestion
        internal static string GetStringClaimValueType(string str, string claimType)
```